src/NativeMessageHost/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ if (FIREBREATH)
 endif()
 
 include_directories(
+    ${FB_GECKOSDK_SOURCE_DIR}
```

Comment:
Hmm; I believe I know why it's used, now that I think it through.  I don't love it, but I guess it'll work for now.